mappings/net/minecraft/client/render/entity/EntityRendererStrayOverlay.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS none/bvz net/minecraft/client/render/entity/EntityRendererStrayOverlay
-	FIELD a OVERLAY_TEXTURE Lnone/kp;
+	FIELD a STRAY Lnone/kp;
```

Comment:
We haven't used the name of the thing so far and I don't think we should start. My vote is for `SKIN`, though `SKIN_OVERLAY` is an alternative.